workbench-core/environments/src/services/environmentTypeConfigService.test.ts

```diff
@@ -2,9 +2,33 @@
  *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  *  SPDX-License-Identifier: Apache-2.0
  */
-
-const envTypeConfigId = '40b01529-0c7f-4609-a1e2-715068da5f0e';
-jest.mock('uuid', () => ({ v4: () => envTypeConfigId }));
+const envTypeId = '1b0502f3-121f-4d63-b03a-44dc756e4c20';
+const envType = {
```

Comment:
Maybe `const envType: EnvironmentType = ...` so we get all the type checking?

Reply:
added in last commit